### PR TITLE
Add Imgix detection

### PR DESCRIPTION
**Pull request by charisTheo**
This PR adds [Imgix](https://imgix.com/) (Digital Asset CDN) detection.

Some example pages that use Imgix for testing:

1. https://www.producthunt.com/posts/omnisearch
2. https://www.kickstarter.com/projects/keyitechnology/meet-loona-the-petbot-you-will-fall-in-love
3. https://imgix.com/

### CODE REVIEW

File: src/technologies/i.json [hunk 1]
```diff
@@ -44,6 +44,24 @@
...
+    ],
+    "description": "Imgix transforms, optimizes, and intelligently caches your entire asset library for faster pages, higher engagement, and a simpler workflow.",
+    "dns": {
+      "SOA": "\\.imgix\\.net"
+    },
+    "dom": "img[src*='.imgix.net/'], img[data-src*='.imgix.net/'], img[srcset*='.imgix.net/'], source[src*='.imgix.net/'], source[data-src*='.imgix.net/']",
+    "icon": "Imgix.svg",
+    "pricing": [
+      "freemium",
+      "low",
```
Comment (nurbek91) [inline src/technologies/i.json:60]: mid instead low

File: src/technologies/i.json [hunk 2]
```diff
@@ -44,6 +44,24 @@
...
+    "cats": [
+      31,
+      95
+    ],
+    "description": "Imgix transforms, optimizes, and intelligently caches your entire asset library for faster pages, higher engagement, and a simpler workflow.",
+    "dns": {
+      "SOA": "\\.imgix\\.net"
+    },
+    "dom": "img[src*='.imgix.net/'], img[data-src*='.imgix.net/'], img[srcset*='.imgix.net/'], source[src*='.imgix.net/'], source[data-src*='.imgix.net/']",
+    "icon": "Imgix.svg",
```
Comment (nurbek91) [inline src/technologies/i.json:57]: Imgix
32x32

File: src/technologies/i.json [hunk 3]
```diff
@@ -44,6 +44,24 @@
     "implies": "Windows Server",
     "website": "http://www.iis.net"
   },
+  "Imgix": {
+    "cats": [
+      31,
+      95
+    ],
+    "description": "Imgix transforms, optimizes, and intelligently caches your entire asset library for faster pages, higher engagement, and a simpler workflow.",
```
Comment (nurbek91) [inline src/technologies/i.json:52]: Imgix is a visual media platform for managing, processing, rendering, optimising and delivering your existing images.

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!!
Need small changes

**Comment by charisTheo** [comment]
Thanks @nurbek91! I've made the changes :)

**Review by nurbek91 — APPROVED** [review]
:heart_hands:

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**